hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/NearCacheTest.java

```diff
@@ -608,16 +713,7 @@ public void testAfterExecuteOnEntriesNearCacheIsInvalidated() {
         populateNearCache(map, mapSize);
 
         EntryObject e = Predicates.newPredicateBuilder().getEntryObject();
-        Predicate predicate = e.get("salary").equal(0);
-        map.executeOnEntries(
-                entry -> {
-                    Employee employee = entry.getValue();
-                    double currentSalary = employee.getSalary();
-                    double newSalary = currentSalary + 10;
-                    employee.setSalary(newSalary);
-                    return newSalary;
-                }, predicate);
-
+        map.executeOnEntries(new TestObjectBasedReadOnlyProcessor());
         assertEquals(0, getNearCacheStats(map).getOwnedEntryCount());
```

Comment:
As far as I see we are missing tests for submitToKeys, executeOnKeys and one of executeOnEntries (there are two executeOnEntries, one of it is with predicate, the other is without predicate.)

Reply:
Added